proto/cmp/types/v1alpha/seat_map.proto

```diff
@@ -141,6 +143,12 @@ message SeatMapInventory {
   repeated SectionInventory sections = 2;
 }
 
+message LocalizedSeatAttributeSet {
```

Comment:
where is this used ?

I think you used LocalizedSeatAttributeSet instead of changing SeatAttribute, but then we will need to use LocalizedSeatAttributeSet instead in some other places.

Reply:
Good catch, I was missing a change in the message Seat. Please check how it is now. 
https://github.com/chain4travel/camino-messenger-protocol/pull/23/commits/42bf29741d850671ee440dd73993e240c5e7c7d0